src/main/scala/info/bethard/anafora/Anafora.scala

```diff
@@ -3,18 +3,20 @@ package info.bethard.anafora
 import com.codecommit.antixml.Elem
 import com.codecommit.antixml.{text => ElemText}
 import com.codecommit.antixml.XML
-
+import com.codecommit.antixml.*
 
 object Data {
   def fromPaths(xmlPath: String, textPath: Option[String]) = apply(
     XML.fromSource(io.Source.fromFile(xmlPath)), textPath.map(p => io.Source.fromFile(p).mkString))
-
-  def apply(xml: Elem, text: Option[String]) = new Data(xml, text)
+  def apply(xml: Elem, text: Option[String]) = text match {
+    case None => new Data(xml, None)
+    case Some(text) => new Data(xml, Some(text.replaceAll("(\r\n)|\r|\n", "\n")))
```

Comment:
Don't make this modification in Anafora.scala. Make it in your scorer. That said, it looks like your scorer no longer uses this at all, so you could just remove this change entirely?